controls/srg_ctr/SRG-APP-000038-CTR-000105.yml

```diff
@@ -9,3 +9,58 @@ controls:
   - configure_network_policies_namespaces
```

Comment:
Can you also add `configure_network_policies` so that we know we are running with the correct SDN?

Reply:
Please remember to add the SRG reference to that rule